_ci/release.sh

```diff
@@ -18,6 +18,8 @@ mvn -B \
     ${DRY_RUN} \
     -Prelease \
     "-Darguments=-Prelease -DskipTests -Dmaven.javadoc.skip -Dadditionalparam=-Xdoclint:none" \
+    -DreleaseVersion="2.1.2-test-0" \
+    -DdevelopmentVersion="2.1.2-SNAPSHOT" \
```

Comment:
I assume we revert this change (ie. should not be merged) ?

Reply:
We should also update pom.xml to 2.2.0-SNAPSHOT, eg.

mvn versions:set -DnewVersion=2.2.0-SNAPSHOT

Reply:
Thanks @PunchedChimera for fixing here: https://github.com/Alfresco/alfresco-transform-core/pull/192/commits/1b95fa367713468a8287dead1d5295d5b7bd6374

Reply:
Removed and versions updated to 2.2.0-SNAPSHOT